tests/units/anta_tests/test_bfd.py

```diff
@@ -414,7 +412,8 @@
         "expected": {
             "result": "failure",
             "messages": [
-                "Following BFD peers are not up:\n192.0.255.7 is down in default VRF with remote disc 0.\n192.0.255.71 is down in MGMT VRF with remote disc 0."
+                "Peer: 192.0.255.7 VRF: default - Session not properly established; State: down Remote Discriminator: 0",
+                "Peer: 192.0.255.71 VRF: MGMT - Session not properly established; State: down Remote Discriminator: 0",
```

Comment:
Why is `down` not capitalized like the previous test `Down`?

Reply:
Yes It should down.
Thanks!! Updated